sh/tests/array-selector_test.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash 
+
+
+# Unofficial Bash Strict Mode
+set -euo pipefail
+IFS='
+'
+
+script_dirS=$(cd -P "$(dirname "$BASH_SOURCE")" &>/dev/null && pwd)
+
+challenge() {
+    submitted=$(bash "$script_dirS"/student/array-selector.sh $1)
+    expected=$(bash "$script_dirS"/solutions/array-selector.sh $1)
+
+    diff <(echo "$submitted") <(echo "$expected")
+}
+
+# Test with numbers - out of range included
+for num in {0..6}
```

Comment:
the for is not working i thin it is because it starts at 0 and 0 is supposed to give you error and it is not working 
If you do the for like so:
```
for num in {1..6}; do
    challenge $num
done
```
it will be a valid test for the correct values of the array, but it will still fail for the error part